### ISSUE TITLE
shadow mode @ wayland client: content drawn to just 1/4th of window size

### ISSUE THREAD

**Issue by thomas725** [Xpra-org/xpra#4255]
**Describe the bug**
I'm connecting from a KDE Plasma Wayland client to a Cinnamon X11 server in shadow mode. The Window size seems reasonable, but the content of the desktop is only drawn to the lower left 1/4th of the window, but the mouse pointer is interpreted as if the whole window represents the servers desktop. So it is impossible to control like that.

**System Information (please complete the following information):**
 - Server OS: Linux Mint 21.3 Kernel 6.2.0-32, Cinnamon Desktop
 - Client OS: Manjaro, Kernel 6.6.32-1 KDE Plasma Wayland Desktop
 - Xpra Server Version v6.0-r0
 - Xpra Client Version v6.0-r0

**Screenshot**
![Screenshot_20240614_093754](https://github.com/Xpra-org/xpra/assets/68635351/0a777763-e49e-485f-b2ce-9e30bc540032)

**Comment by thomas725** [comment]
I've recently upgraded the server from v3.1 but saw those warnings during the upgrade:
```
dpkg: warning: unable to delete old directory '/usr/lib/udev/rules.d': Directory not empty
dpkg: warning: unable to delete old directory '/etc/xpra/conf.d': Directory not empty
dpkg: warning: unable to delete old directory '/etc/xpra': Directory not empty
```
Is that maybe related to my new issue?

**Comment by totaam** [comment]
> Is that maybe related to my new issue?

No. This is `dpkg` struggling to remove the old package - for whatever reason.

To workaround the window size issue, you can run your client with `--opengl=no`.
It will be much slower, but should draw over the whole window.

**Comment by thomas725** [comment]
thanks, that works! but as predicted, it's quite slow. but at least it works ^^
thank you very much :)

**Comment by totaam** [comment]
This should have been fixed in 6.1.x, please comment if not.
